java/client/src/main/java/glide/api/RedisClusterClient.java

```diff
@@ -68,4 +73,35 @@ public CompletableFuture<String> ping(@NonNull String str, @NonNull Route route)
         return commandManager.submitNewCommand(
                 Ping, new String[] {str}, route, this::handleStringResponse);
     }
+
+    @Override
+    public CompletableFuture<ClusterValue<String>> info() {
+        return commandManager.submitNewCommand(
+                Info, new String[0], response -> ClusterValue.of(handleObjectOrNullResponse(response)));
```

Comment:
Is null accepted here?
```suggestion
                Info, new String[0], response -> ClusterValue.of(handleObjectResponse(response)));
```

Reply:
I suppose that makes sense.